src/bin/utils/utils.ts

```diff
@@ -112,7 +112,7 @@ function outputFormatter(msg: OutputObject): string | Uint8Array {
     let currError = msg.data;
     let indent = '  ';
     while (currError != null) {
-      if (currError instanceof grpcErrors.ErrorPolykeyRemote) {
+      if (currError instanceof grpcErrors.ErrorPolykeyRemoteOLD) {
```

Comment:
Don't we need to have something like this but for the new JSON based RPC?

Reply:
We do have it. it's called `ErrorPolykeyRemote`. I had to rename the old one to avoid conflict until it's fully removed.

Reply:
I know I meant the line itself. You're checking for the remote old, why not check for the `ErrorPolykeyRemote` too?

Reply:
Ah I see, I misunderstood. It should be checking for both until the old one is removed. I'll fix this up.